src/main/java/seedu/duke/command/DeleteCommand.java

```diff
@@ -36,6 +39,7 @@ public class DeleteCommand extends Command {
 
     // The optional tags that may exist in the user input
 
+    private static final Logger logging = Logger.getLogger(DeleteCommand.class.getName());
```

Comment:
Suggest that you might want to set the variable names to be consistent in future (e.g. your `logging` and `theLogger` for delete and parser respectively can be `deleteLogger` and `purgeLogger`?